packages/dds/tree/src/test/simple-tree/utils.ts

```diff
@@ -63,3 +67,27 @@ export function pretty(arg: unknown): number | string {
 	}
 	return JSON.stringify(arg);
 }
+
+/**
+ * Creates a branch of the input tree view and returns a new tree view for the branch.
+ *
+ * @remarks To merge the branch back into the original view after applying changes on the branch view, use
+ * `<originalView>.checkout.merge(<branchView>.checkout)`.
+ *
+ * @param originalView - The tree view to branch.
+ * @returns A new tree view for a branch of the input tree view, and an {@link ITreeCheckoutFork} object that can be
+ * used to merge the branch back into the original view.
+ */
+export function getViewForForkedBranch<TSchema extends ImplicitFieldSchema>(
+	originalView: SchematizingSimpleTreeView<TSchema>,
+): { forkView: SchematizingSimpleTreeView<TSchema>; forkCheckout: ITreeCheckoutFork } {
```

Comment:
Should we define a type for this? That would give us a better place to document it properties and such.